receiver/filelogreceiver/README.md

```diff
@@ -180,7 +180,130 @@ Exception in thread 2 "main" java.lang.NullPointerException
 
 ## Offset tracking
 
-`storage` setting allows to define the proper storage extension to be used for storing file offsets. 
+`storage` setting allows to define the proper storage extension to be used for storing file offsets.
 While the storage parameter can ensure that log files are consumed accurately, it is possible that
 logs are dropped while moving downstream through other components in the collector.
 For additional resiliency, see [Fault tolerant log collection example](../../examples/fault-tolerant-logs-collection/README.md)
+
+### File storage
+
+A common storage extension that's used for tracking log file offsets is the
+[`filestorage` extension](../../extension/storage/filestorage). Sometimes, typically for debugging reasons, it's useful
+to take a peek at the file in which offsets are stored. At the moment, the simplest way to do this is by printing out
+the contents of this file using the `strings` utility.
+
+Consider a collector pipeline that's using the `filelog` receiver with the `storage` extension as shown
+below. Note that [compaction](../../extension/storage/filestorage/README.md#compaction) is not being used.
+
+```yaml
+receivers:
+  filelog:
+    include: /tmp/*.log
+    storage: file_storage/filelogreceiver
+
+exporters:
+  ...
+
+extensions:
+  file_storage/filelogreceiver:
+    directory: /tmp/otelcol/file_storage/filelogreceiver
+
+service:
+  extensions: [file_storage/filelogreceiver]
+  pipelines:
+    logs:
+      receivers: [filelog]
+      exporters: [...]
+```
+
+Assume there are no log files matching `/tmp/*.log` when the above collector pipeline starts executing. In this
+scenario, the `/tmp/otelcol/file_storage/filelogreceiver` directory contains one file:
+
+```
+$ ls /tmp/otelcol/file_storage/filelogreceiver
+receiver_filelog_
+```
+
+This is a binary file, so we can read its contents using the `strings` utility.
+
+```
+$ strings /tmp/otelcol/file_storage/filelogreceiver/receiver_filelog_
+default
+file_input.knownFiles0
+default
+file_input.knownFiles0
+default
+file_input.knownFiles0
```

Comment:
@djaglowski Do you happen to know why the entries in the `filestorage` file are repeated 3 times? Best that I can tell, this has to do with how bbolt is storing the data on disk. I'd like to mention the reason here in these docs so users don't get confused by the repeated entries.

Reply:
Interesting observation. I don't see any obvious reason from the fileconsumer side. Might be a bbolt peculiarity or a real problem with the storage extension or interface.